scripts/fixture_local_test_setup.sh

```diff
@@ -423,27 +423,29 @@ env \
   --hopr-amount "0.0"
 # }}}
 
-log "Waiting for port binding"
-
-#  --- Wait for ports to be bound --- {{{
-wait_for_regex "${node1_log}" "STARTED NODE"
-wait_for_regex "${node2_log}" "STARTED NODE"
-wait_for_regex "${node3_log}" "STARTED NODE"
-wait_for_regex "${node4_log}" "STARTED NODE"
-wait_for_regex "${node5_log}" "STARTED NODE"
-wait_for_regex "${node6_log}" "STARTED NODE"
-wait_for_port 19096 "127.0.0.1" "${node6_log}"
-wait_for_regex "${node7_log}" "STARTED NODE"
-# }}}
+log "Waiting for nodes to become ready"
+# $1 = node api endpoint
+wait_for_api_ready() {
+  local api_port=${1}
+  local api_token=${2}
 
-log "Sleep for 30 seconds to ensure announcements are confirmed on-chain"
-sleep 30
+  curl -X GET -m 3 --connect-timeout 300 -s -H X-Auth-Token:${api_token} -H Content-Type:application/json --url http://127.0.0.1:${api_port}/api/v3/readyz -o /dev/null -w %{http_code} -d
```

Comment:
Why not use our curl helper function?

Reply:
So that there are as little conflicts with the fixture removal PR as possible.

Reply:
Used the api.sh